### PR TITLE
feat: update mainnet deployment

### PR DESCRIPTION
**Pull request by karlfloersch**
**Description**
Add a new mainnet deployment of v0.3.0.

### CODE REVIEW

**Comment by tynes** [comment]
Can you regenerate the `README`? This will be good to merge after that

File: packages/contracts/deployments/mainnet/OVM_CanonicalTransactionChain.json [hunk 1]
```diff
@@ -601,32 +601,32 @@
...
+    "cumulativeGasUsed": "8803160",
     "status": 1,
     "byzantium": true
   },
   "args": [
-    "0xd3EeD86464Ff13B4BFD81a3bB1e753b7ceBA3A39",
-    18000000,
-    1200000,
+    "0x668E5b997b9aE88a56cd40409119d4Db9e6d752E",
+    15769999,
```
Comment (smartcontracts) [inline packages/contracts/deployments/mainnet/OVM_CanonicalTransactionChain.json:622]: Why these weird config values?

File: packages/contracts/deployments/mainnet/OVM_ExecutionManager.json [hunk 2]
```diff
@@ -670,38 +671,38 @@
...
+    "cumulativeGasUsed": "10740618",
     "status": 1,
     "byzantium": true
   },
   "args": [
-    "0xd3EeD86464Ff13B4BFD81a3bB1e753b7ceBA3A39",
+    "0x668E5b997b9aE88a56cd40409119d4Db9e6d752E",
     {
-      "minTransactionGasLimit": 50000,
+      "minTransactionGasLimit": 100000,
```
Comment (smartcontracts) [inline packages/contracts/deployments/mainnet/OVM_ExecutionManager.json]: Can we double check that there are no problems with 100k min

Comment (karlfloersch) [reply packages/contracts/deployments/mainnet/OVM_ExecutionManager.json]: @ben-chain check me on this one, but I think that 100k should be fine considering we have this check in the deposits: https://github.com/ethereum-optimism/optimism/blob/52d9e6b73362a203bdae3fa0182bd6e7c9693b13/packages/contracts/contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol#L41

File: packages/contracts/deployments/mainnet/OVM_CanonicalTransactionChain.json (hunk 1 above)
Comment (karlfloersch) [reply packages/contracts/deployments/mainnet/OVM_CanonicalTransactionChain.json:622]: Oh damn thought I changed that to `15770000` which is ~half a year.

File: packages/contracts/deployments/mainnet/OVM_ChainStorageContainer:CTC:batches.json [hunk 3]
```diff
@@ -0,0 +1,453 @@
+{
```
[the remark this answers is not in the document]
Comment (karlfloersch) [reply packages/contracts/deployments/mainnet/OVM_ChainStorageContainer:CTC:batches.json:1]: Ah what is the context for that? I noticed they were hyphens in the old mainnet deploy but didn't know why we did that. I'll add a commit to fix that though

[the remark this answers is not in the document]
Comment (karlfloersch) [reply packages/contracts/deployments/mainnet/OVM_ChainStorageContainer:CTC:batches.json:1]: I updated the names of the files to use hyphens. Anything else that needs to change?

File: packages/contracts/deployments/mainnet/OVM_ExecutionManager.json (hunk 2 above)
Comment (ben-chain) [reply packages/contracts/deployments/mainnet/OVM_ExecutionManager.json:693]: Yeah, these should be combined and always kept the same.  I agree that this is OK due to the current require that Karl pointed out.  One note is it [will make things OOG more often](https://github.com/ethereum-optimism/optimism/blob/52d9e6b73362a203bdae3fa0182bd6e7c9693b13/packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol#L209), though.

**Comment by platocrat** [comment]
^we had to bring the branch up-to-date with `develop` again, so I went ahead and merged to do so. 
Still needs a review

**Comment by tynes** [comment]
Merging this because its high priority and there is no going back at this point. The config options are safe but we should consider discussing them in public more next time so that there can be more input

**Pull request merged**